judgels-frontends/raphael/package.json

```diff
@@ -56,7 +57,9 @@
     "lint:prettier": "prettier --list-different 'src/**/*.ts' 'src/**/*.tsx'",
     "format:prettier": "prettier --write 'src/**/*.ts' 'src/**/*.tsx'",
     "test:ts": "TZ=Asia/Jakarta react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "postinstall": "npm-run-all prepare:mathjax",
```

Comment:
No need for `npm-run-all` for only 1 command

Reply:
Okay, changed.